src/main/webapp/resources/js/components/samples/components/SampleFiles.tsx

```diff
@@ -93,9 +93,12 @@ export function SampleFiles() {
    */
```

Comment:
Can you check the dependencies for the above useEffect

Reply:
As per our discussion this useEffect only has the one dependency